src/main/resources/application.properties

```diff
@@ -1,2 +1,6 @@
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
 spring.datasource.driver-class-name=org.postgresql.Driver
+
+spring.cache.redis.cache-null-values=false
+# 24 hours
+spring.cache.redis.time-to-live=1440000
```

Comment:
if we dont cache by default, how do we use redis?

isnt 
```properties
spring.cache.type=redis
```
missing?

Reply:
Yet another spring boot thing. I have `spring-boot-starter-data-redis` as a dependency, and I guess spring detects it as the provider. And caching is declaratively enabled in the `RedisCacheConfiguration` class. In the [official spring boot documentation ](https://docs.spring.io/spring-boot/docs/2.1.6.RELEASE/reference/html/boot-features-caching.html#boot-features-caching-provider)(see the first remark in that section) is advised to set this property when you want to disable the caching(which I am doing the tests properties)